File: SCC0600/exercíciosPy/ex20.py
import math

largura = float(input())
altura = float(input())
custo = float(input())
rendimento = float(input())

area = largura * altura
latas = math.ceil(area / rendimento)
custototal = latas * custo

print(latas)
print('{:.2f}'.format(custototal))

File: SCC0600/exerciciosC/ex20.c
#include <stdio.h>
#include <math.h>

int main (void) {
    float largura; float altura; float custo; float rendimento;
    scanf("%f", &largura);
    scanf("%f", &altura);
    scanf("%f", &custo);
    scanf("%f", &rendimento);

    int area = largura * altura;
    int latas = ceil(area / rendimento);
    float total = latas * custo;

    printf("%d\n", latas);
    printf("%.2f\n", total);
}